src/utils/config-validator.ts

```
@@ -0,0 +1,49 @@
+import { ExtensionConfig, LogLevel } from '../types/extension';
+import  { type ILogger } from '../types/extension';
```

ESLint

[error] '../types/extension' import is duplicated.

(no-duplicate-imports)